components/misc/stringops.hpp

```diff
@@ -18,6 +17,19 @@ class StringUtils
         }
     };
 
+    // Allow to convert complex arguments to C-style strings for format() function
+    template <typename T>
+    static T Argument(T value) noexcept
```

Comment:
Why east const here and below but not `char const*`? Also function name should start from lower case by our [conventions](https://wiki.openmw.org/index.php?title=Code_Formatting_Conventions).

Reply:
> Why east const here and below but not char const*? 

I do not understand what do you mean.

Reply:
> Also function name should start from lower case by our conventions.

Resolved.

Also if you mean that we should use the `char const*` instead of `const char*`, the `const char*` is more widely used in OpenMW. If we want to change it, it should be changed over the whole engine, but it is outside of scope of this PR.